easy-ui-react/src/styles/_scrollbars.scss

```diff
@@ -10,9 +10,13 @@
   --os-handle-bg-active: #{token-helpers.design-token("color.gray.500")};
   --os-track-border-radius: 999px;
   --os-handle-border-radius: 999px;
+
+  // there's a bug in Firefox with overlay elements rendering below the
+  // scrollbar due to `contain` rasterizing this slice of the screen
+  contain: unset;
 }
 
-.ezui-os-theme-overlay {
+.os-scrollbar.ezui-os-theme-overlay {
```

Comment:
sometimes there will be other os scrollbars on the page (i.e. storybook) that conflict with our theme, this remedies that